### PR TITLE
Add auto deploy using chrome-webstore-upload-cli package

### PR DESCRIPTION
**Pull request by paulmolluzzo**
This is an alternative to #25 that uses [chrome-webstore-upload-cli](https://github.com/DrewML/chrome-webstore-upload-cli) to auto-deploy the extension to the chrome webstore for tagged releases.

To get this working we need to set some environment variables for Travis CI. You can do this by adding encrypted credentials to the .travis.yml file, or through the Travis CI website. I documented it a bit here https://github.com/paulmolluzzo/test-deploy-chrome

H/T to @DrewML for the deployment package and helping me get this figured out.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,11 @@
 {
   "scripts": {
+    "zip": "zip -r extension.zip extension/",
```
Comment (sindresorhus) [inline package.json]: Don't really need a separate script for zipping. Just put it inline.

File: package.json [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+    "release": "npm run zip && webstore upload --file extension.zip --extension-id 'nlfgmdembofgodcemomfeimamihoknip' --auto-publish",
```
Comment (sindresorhus) [inline package.json]:
```
--file=extension.zip
```

Comment (sindresorhus) [inline package.json]:
```
--extension-id=nlfgmdembofgodcemomfeimamihoknip
```

Comment (sindresorhus) [inline package.json]: @DrewML Would be nice if your module handled the zipping part too.

Comment (DrewML) [inline package.json]: https://github.com/DrewML/chrome-webstore-upload-cli/issues/2 Makes sense. Will probably end up doing it tomorrow night. We can either wait on this PR, or I can submit an updated PR when it's changed

Comment (DrewML) [inline package.json]: @paulmolluzzo Just published `1.0.0`. `--source` is used now instead of `--file`, and you can just pass it the directory that `manifest.json` is in. Everything else should be able to stay the same with regards to the setup.

Comment (paulmolluzzo) [inline package.json]: @DrewML super rad! Of course I also took out the `zip` command stuff as that's no longer needed.

**Comment by paulmolluzzo** [comment]
Are we OK for getting this into `master`? Were the Google keys added to Travis CI?

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 node_modules
+extension.zip
```
Comment (sindresorhus) [inline .gitignore]: Why is this needed?

File: package.json [hunk 4]
```diff
@@ -1,8 +1,10 @@
 {
   "scripts": {
+    "release": "webstore upload --source=extension/ --extension-id=nlfgmdembofgodcemomfeimamihoknip --auto-publish",
```
Comment (sindresorhus) [inline package.json]: `--source=extension/` => `--source=extension`

**Comment by paulmolluzzo** [comment]
@sindresorhus both fixed.

**Comment by sindresorhus** [comment]
Alright. Let's try this out.

**Comment by sindresorhus** [comment]
Auto-deploy is working fine :)
